### ISSUE TITLE
Improve validation error experience for screen reader users

### ISSUE THREAD

**Issue by notlee** [Financial-Times/o-forms#174]
Suggestion by @benbarnett 

>For visually impaired users, 'rolling' (blur) validation can easily be missed when tabbing through a form. Perhaps something like:
>
>When the form is submitted...
>If the validation passes, the submission carries on as normal
>if the validation fails, an error summary is appended to the top of the form, and it is given focus to alert screen-reader users

Related Issue: https://github.com/Financial-Times/o-forms/issues/152

**Comment by notlee** [comment]
`aria-describedby` may be helpful to improve this experience:
 https://github.com/Financial-Times/o-forms/pull/205#discussion_r181650719
